utils/rpms/daos.spec

```diff
@@ -226,12 +226,21 @@ This is the package needed to run a DAOS client
 
 %package tests
 Summary: The entire DAOS test suite
-Requires: %{name}-client-tests-openmpi%{?_isa} = %{version}-%{release}
-Requires: %{name}-server-tests-openmpi%{?_isa} = %{version}-%{release}
+Requires: %{name}-client-tests%{?_isa} = %{version}-%{release}
+Requires: %{name}-server-tests%{?_isa} = %{version}-%{release}
```

Comment:
I'm not sure that you can do this all in one commit.  This will cause testing on this branch to break until the pipeline-lib change lands.  Unless you close landing on this branch after this until the pipeline-lib update lands.